src/datumaro/plugins/transforms.py

```diff
@@ -266,6 +266,31 @@ def convert_bbox(bbox, img_h, img_w):
         )
 
 
+class BoxesToPolygons(ItemTransform, CliPlugin):
+    def transform_item(self, item):
+        annotations = []
+        for ann in item.annotations:
+            if ann.type == AnnotationType.bbox:
+                if not isinstance(item.media, Image):
+                    raise Exception("Image info is required for this transform")
+                annotations.append(self.convert_bbox(ann))
+            else:
+                annotations.append(ann)
```

Comment:
```suggestion
        annotations = [
             self.convert_bbox(ann) if ann.type == AnnotationType.bbox else ann
             for ann in item.annotations
        ]
```

Reply:
Oh, you're right. We don't need (w,h) of image for conversion, so, media type condition isn't needed.